sdk/python/tests/integration/registration/test_registry.py

```diff
@@ -287,7 +287,7 @@ def odfv1(feature_df: pd.DataFrame) -> pd.DataFrame:
             Feature(name="odfv1_my_feature_1", dtype=ValueType.FLOAT),
             Feature(name="odfv1_my_feature_2", dtype=ValueType.INT32),
         ],
-        sources={"request_source": request_source},
+        sources=[request_source],
```

Comment:
Do we have a test that tests the `Dict[str, Source]` version to make sure that the changes are backwards compatibile?